src/Search.js

```diff
@@ -26,4 +27,8 @@ const Search = () => {
   );
 };
 
+function SearchButton() {
+  return <button className="btn btn-primary">Search</button>;
+}
+
```

Comment:
like mentioned above, extract this out into it's own component :) then anywhere in your app you needed a search button you could import it in